addon/mixins/touch-action.js

```diff
@@ -37,15 +36,15 @@ export default Mixin.create({
   ignoreTouchAction: false,
 
   init() {
-    this._super();
+    this._super(...arguments);
 
     const {
       tagName,
       ignoreTouchAction,
       click
     } = this;
 
-    const hasClick = click !== K;
+    const hasClick = typeof click === 'function';
```

Comment:
(also `null/undefined+''.length > 0`  is better than typeof tagName check too then yeah? Only would need to worry about if it was an array []+'' === '' but that would be user error - should i add that optimization too?